commit c31b42c143250be89c6fc9817bf52d3293a28a05
Author: Bob Peterson <rpeterso@redhat.com>
Date:   Fri Jan 6 17:01:57 2012 -0600

    Change man pages and gfs2_convert messages to include GFS
    
    Since fsck.gfs2 can now handle GFS1 file systems, this patch changes
    the message in gfs2_convert and the man pages to reflect that.
    
    rhbz#675723

diff --git a/gfs2/convert/gfs2_convert.c b/gfs2/convert/gfs2_convert.c
index 8ed2c1a..fa6461f 100644
--- a/gfs2/convert/gfs2_convert.c
+++ b/gfs2/convert/gfs2_convert.c
@@ -1604,7 +1604,7 @@ static void give_warning(void)
 	printf("WARNING: This can't be undone.  It is strongly advised "	\
 		   "that you:\n\n");
 	printf("   1. Back up your entire filesystem first.\n");
-	printf("   2. Run gfs_fsck first to ensure filesystem integrity.\n");
+	printf("   2. Run fsck.gfs2 first to ensure filesystem integrity.\n");
 	printf("   3. Make sure the filesystem is NOT mounted from any node.\n");
 	printf("   4. Make sure you have the latest software versions.\n");
 }/* give_warning */
diff --git a/gfs2/man/fsck.gfs2.8 b/gfs2/man/fsck.gfs2.8
index 462e918..3ecfbe3 100644
--- a/gfs2/man/fsck.gfs2.8
+++ b/gfs2/man/fsck.gfs2.8
@@ -1,18 +1,19 @@
 .TH fsck.gfs2 8
 
 .SH NAME
-fsck.gfs2 - Offline GFS2 file system checker
+fsck.gfs2 - Offline GFS and GFS2 file system checker
 
 .SH SYNOPSIS
 .B fsck.gfs2
 [\fIOPTION\fR]... \fIDEVICE\fR
 
 .SH WARNING
-All GFS2 nodes \fImust\fP have the GFS2 filesystem unmounted before running
-fsck.gfs2.  Failure to unmount all nodes may result in filesystem corruption.
+All computers \fImust\fP have the filesystem unmounted before running
+fsck.gfs2.  Failure to unmount from all nodes in a cluster will likely result
+in filesystem corruption.
 
 .SH DESCRIPTION
-fsck.gfs2 will check that the GFS2 file system on a device is structurally valid.
+fsck.gfs2 will check that the GFS or GFS2 file system on a device is structurally valid.
 It should not be run on a mounted file system.  If file system corruption is
 detected, it will attempt to repair the file system.  There is a limit to what
 fsck.gfs2 can do.  If important file system structures are destroyed, such that
diff --git a/gfs2/man/gfs2_convert.8 b/gfs2/man/gfs2_convert.8
index 5969432..ca0f64f 100644
--- a/gfs2/man/gfs2_convert.8
+++ b/gfs2/man/gfs2_convert.8
@@ -10,7 +10,7 @@ gfs2_convert - Convert a GFS1 filesystem to GFS2
 .SH DESCRIPTION
 gfs2_convert is used to convert a filesystem from GFS1 to GFS2. It is
 required that the GFS1 filesystem be checked and fixed for errors using 
-\fBgfs_fsck\fP and that the filesystem be backed up before
+\fBfsck.gfs2\fP and that the filesystem be backed up before
 attempting to convert it. The convert process is irreversible and any
 error encountered during the conversion can result in the abrupt
 termination of the program and consequently an unusable filesystem.
